pkg/fqdn/bootstrap/cell.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package bootstrap
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/cilium/hive/cell"
+
+	"github.com/cilium/cilium/pkg/endpointmanager"
+	"github.com/cilium/cilium/pkg/fqdn/defaultdns"
+	"github.com/cilium/cilium/pkg/fqdn/namemanager"
+	"github.com/cilium/cilium/pkg/ipcache"
+	"github.com/cilium/cilium/pkg/policy"
+	"github.com/cilium/cilium/pkg/proxy"
+	"github.com/cilium/cilium/pkg/proxy/accesslog"
+)
+
+// Cell provides the FQDN bootstrap functionality
+var Cell = cell.Module(
+	"fqdn-bootstrap",
+	"Cell provides the FQDN bootstrap functionality",
```

Comment:
nit: maybe more like `Bootstraps the FQDN policy subsystem`?

Reply:
thanks, changed